include/measurement_kit/http/http.hpp

```diff
@@ -77,17 +98,34 @@ void request_connect(Settings, Callback<Error, Var<net::Transport>>,
                      Var<Logger> = Logger::global());
 
 void request_send(Var<net::Transport>, Settings, Headers, std::string,
-        Callback<Error>);
+                  Callback<Error>);
 
 void request_recv_response(Var<net::Transport>, Callback<Error, Var<Response>>,
-        Var<Reactor> = Reactor::global(), Var<Logger> = Logger::global());
+                           Var<Reactor> = Reactor::global(),
+                           Var<Logger> = Logger::global());
 
 void request_sendrecv(Var<net::Transport>, Settings, Headers, std::string,
-        Callback<Error, Var<Response>>, Var<Reactor> = Reactor::global(),
-        Var<Logger> = Logger::global());
+                      Callback<Error, Var<Response>>,
+                      Var<Reactor> = Reactor::global(),
+                      Var<Logger> = Logger::global());
+
+/*
+ * For settings the following options are defined:
+ *
+ *     {
+ *       {"http/follow_redirects", boolean},
+ *       {"http/url", std::string},
+ *       {"http/ignore_body", boolean},
+ *       {"http/method", "GET|DELETE|PUT|POST|HEAD|..."},
+ *       {"http/http_version", "HTTP/1.1"},
+ *       {"http/path", by default is taken from the url}
+ *     }
+ *
+ * Currently `http/follow_redirects` is not supported.
+ */
```

Comment:
This commented was along with `RequestSerialized`. Moved it here where it's most likely to inform users.